api-catalog-ui/frontend/src/components/ServiceVersionDiff/ServiceVersionDiff.jsx

```diff
@@ -21,29 +21,30 @@ import {
     DialogActions,
     Divider,
 } from '@material-ui/core';
+import PropTypes from 'prop-types';
 import CloseIcon from '@material-ui/icons/Close';
 
 export default class ServiceVersionDiff extends Component {
     constructor(props) {
-        const { version1, version2, selectedVersion } = props;
+        const { versions } = props;
         super(props);
         this.state = {
-            selectedVersion1: version1 ? { text: version1 } : undefined,
-            selectedVersion2: version2 ? { text: version2 } : undefined,
+            selectedVersion1: versions?.length ? versions[versions.length - 2] : undefined,
```

Comment:
The `ServiceVersionDiff` component will be called only in case the versions is >= 2, so there won't be any situation where it will get triggered when length is 1.